app/views/options.html

```diff
@@ -9,6 +9,13 @@
     </label>
     <p>Show more detailed numbers for each of your units.</p>
   </div>
+  <div class="form-group">
+    <label class="control-label">
+      <span class="glyphicon glyphicon-eye-open"></span> Show accurate times 
+      <input type="checkbox" ng-init="form.showaccuratetiming=options.showAccurateTiming()"     ng-model="form.showaccuratetiming"   ng-change="options.showAccurateTiming(form.showaccuratetiming)">
+    </label>
+    <p>Show more accurate time estimates.</p>
```

Comment:
If this stays optional, an example like number formatting has would be nice here.